components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -18,16 +17,53 @@ import Image from 'next/image';
  * @param props.isPickSet - if true, the team logo of the picked team shows up on the LeagueEntries card and the button changes from "make a pick" to "chagne pick"
  * @param props.userPickHistory - the user's pick history for this entry
  * @param props.selectedTeamLogo - the team logo
+ * @param props.lockout - if true, the user is locked out from making a pick
  * @returns {React.JSX.Element} - A div element that contains the user's entry information
  */
+
+/**
+ * Display all entries for a league.
+ * @param {string} leagueId - The league id.
+ * @returns {JSX.Element} The rendered entries component.
+ */
 const LeagueEntries = ({
   entryName,
-  linkUrl,
   isEliminated = false,
   isPickSet = false,
+  linkUrl,
   userPickHistory,
   selectedTeamLogo = '',
 }: ILeagueEntriesProps): JSX.Element => {
+  const [isLockedOut, setLockedOut] = useState<boolean>(false);
+
+  useEffect(() => {
+    /**
+     * Checks if the user is locked out from making a pick.
+     */
+    const checkLockout = (): void => {
```

Comment:
This is exactly the same code as the one in Week.tsx. When this occurs, it's a good idea to move this to a shared helper function.